pkg/collect/secret.go

```diff
@@ -60,7 +60,16 @@ func Secret(ctx *Context, secretCollector *troubleshootv1beta1.Secret) ([]byte,
 	return b, nil
 }
 
-func secret(client *kubernetes.Clientset, secretCollector *troubleshootv1beta1.Secret) (*FoundSecret, []byte, error) {
+func secret(client *kubernetes.Clientset, secretCollector *troubleshootv1beta1.Secret) (string, []byte, error) {
+	ns := secretCollector.Namespace
+	if ns == "" {
+		ns = "current-namespace"
```

Comment:
i think this should be empty.  this value will raise questions.

Reply:
...I should be using filepath.join, too